Source/Prism/Common/ParametersExtensions.cs

```diff
@@ -72,6 +76,8 @@ public static IEnumerable<T> GetValues<T>(this IEnumerable<KeyValuePair<string,
                         values.Add((T)kvp.Value);
                     else if (typeof(T).IsAssignableFrom(kvp.Value.GetType()))
                         values.Add((T)kvp.Value);
+                    else if (type.IsEnum && Enum.TryParse<T>(kvp.Value, true, out var enumValue))
```

Comment:
this should be typeof(T).IsEnum... as is this will not compile... also since this would cause us to evaluate typeof(T) for more than one case in loop this should probably be refactored to set `var type = typeof(T)` outside of the loop so we only get the type once.

Reply:
done. I figure out the needs to change on the implmentation:
a ) change Enum.TryParse<T> to Enum.Parse because the first one applies restrictions that T must be inherit from struct
b) remove the ignoreCase parameter to avoid any side effect that can cause, so this will ensure that the user must pass the parameter exactly the name of the enum desired.